### DISCUSSION TITLE
I don't understand how to work with the device

### DISCUSSION THREAD

**Discussion by somethingintheway1** [danielpaulus/go-ios#537] [Q&A]
I threw wintul.dll in system32, I wrote go-ios tunnel start, and I get go-ios tunnel start in response

{"level":"warning","msg":"go-ios agent is not running. You might need to start it with 'ios tunnel start' for ios17+. Use ENABLE_GO_IOS_AGENT=yes for experimental daemon mode.","time":"2024-12-20T23:57:20+07:00"}

{"level":"info","msg":"Tunnel server started","time":"2024-12-20T23:57:50+07:00"}
{"error":"UpdateTunnels: failed to get list of devices: Failed getting devicelist: Error 'unexpected EOF' while reading usbmux package. Only 253 bytes received instead of 1977109506","level":"warning","msg":"failed to update tunnels","time":"2024-12-20T23:58:20+07:00"}

i use a iphone 11, 18 ios, lightning - type C cable

**Comment by shamanec** [comment]
Have you installed iTunes?
